TeamSnapSDK/SDK/DataTypes/Lineups/TSDKEventLineup.m

```diff
@@ -0,0 +1,104 @@
+//
+//  TSDKEventLineup.m
+//  TeamSnapSDK
+//
+//  Created by Jason Rahaim on 4/10/18.
+//  Copyright © 2018 teamsnap. All rights reserved.
+//
+
+#import "TSDKEventLineup.h"
+#import "TSDKEventLineupEntry.h"
+
+@implementation TSDKEventLineup
+
+@dynamic eventId, isPublished, entriesCount, createdAt, updatedAt, notifyTeam, linkEventLineupEntries;
+
++ (void)updateEventLineup:(TSDKEventLineup *)lineup withLineupEntries:(NSArray <TSDKEventLineupEntry *> *)lineupEntries withConfiguration:(TSDKRequestConfiguration *_Nullable)configuration completion:(TSDKEventLineupEntryArrayCompletionBlock _Nullable)completion {
+    TSDKCollectionCommand *command = [TSDKEventLineupEntry commandForKey:@"bulk_update_event_lineup_entries"];
+    
+    if (command == nil) {
+        if (completion != nil) {
+            completion(NO, YES, nil, nil);
+        }
+        return;
+    }
+    
+    NSMutableArray *lineupEntriesArray = [NSMutableArray array];
+    for (TSDKEventLineupEntry *lineupEntry in lineupEntries) {
+        NSMutableArray *lineupProperties = [NSMutableArray array];
+        
+        for (NSString *key in @[@"member_id", @"sequence", @"label"]) {
+            NSString *value = [lineupEntry.collection.data valueForKey:key];
+            if (value == nil) {
+                value = @"";
+            }
+            
+            NSDictionary *lineupProperty = @{@"name" : key,
+                                             @"value" : value };
+            [lineupProperties addObject:lineupProperty];
+        }
+        
+        NSDictionary *lineupProperty = @{@"name" : @"event_lineup_id",
+                                         @"value" : lineup.objectIdentifier };
+        [lineupProperties addObject:lineupProperty];
+        
+        NSDictionary *entryDict = @{@"data":lineupProperties};
+        [lineupEntriesArray addObject:entryDict];
+    }
+    
+    NSError *error = nil;
+    NSData *data = [NSJSONSerialization dataWithJSONObject:lineupEntriesArray options:0 error:&error];
+    if (data != nil) {
+        NSString *commandString = [[[NSString alloc] initWithData:data encoding:NSUTF8StringEncoding] stringByAddingPercentEncodingWithAllowedCharacters:[NSCharacterSet URLQueryAllowedCharacterSet]];
+        NSString *templatesString = [NSString stringWithFormat:@"?templates=%@", commandString];
+        command.href = [command.href stringByAppendingString:templatesString];
+    }
+    
+    [command.data removeObjectForKey:@"templates"];
```

Comment:
The default command `data` dictionary has an empty string stored for `templates`, which v3 really doesn't like and will throw a 400 if you send it in the body payload: `{"templates":""}`